frontend/app/dashboard/src/views/webshops/WebshopsMenu.vue

```diff
@@ -0,0 +1,263 @@
+  () => $context.value?.organizationAuth.hasFullAccess() ?? false
+);
+
+const allWebshops = computed(() => {
+  if (!$organization.value) return [];
+  return $organization.value.webshops;
+});
+
+const visibleWebshops = computed(() =>
+  allWebshops.value
+    .filter((webshop) => webshop.meta.status !== WebshopStatus.Archived)
+    .sort((a, b) =>
+      Sorter.stack(
+        Sorter.byBooleanValue(b.isClosed(), a.isClosed()),
+        Sorter.byStringValue(a.meta.name, b.meta.name)
+      )
+    )
+);
+
+const canCreateWebshops = computed(() =>
+  hasAccessRight(AccessRight.OrganizationCreateWebshops)
+);
+
+const hasWebshopArchive = computed(() =>
+  allWebshops.value.some(
+    (webshop) => webshop.meta.status == WebshopStatus.Archived
+  )
+);
+//#endregion
+
+//#region lifecycle
+onActivated(() => {
+  urlHelpers.setTitle("Verkoop");
+});
+//#endregion
+
+//#region functions
+function hasAccessRight(right: AccessRight) {
+  return $context.value.organizationPermissions?.hasAccessRight(right) ?? false;
+}
+
+function selectButton(button: Button | string) {
+  currentlySelected.value = button;
+}
+
+function isSelected(button: Button | string) {
+  return currentlySelected.value === button;
+}
+
+async function addWebshop() {
+  $navigate(Routes.AddWebshop);
+}
+
+async function openWebshop(webshop: WebshopPreview) {
+  $navigate(Routes.Webshop, { properties: { preview: webshop } });
+}
+
+function selectWebshop(webshop: WebshopPreview) {
+  const id = webshop.id;
+  selectedWebshop.value = id;
+  selectButton("webshop-" + id);
+}
+
+function isWebshopOpen(webshop: WebshopPreview) {
+  return !webshop.isClosed();
+}
+
+//#endregion
+
+//#region routes
+enum Routes {
+  Webshop = "webshop",
+  AddWebshop = "addWebshop",
+  Archive = "archive",
+}
+
+defineRoutes([
+  // webshop detail
+  {
+    url: "@slug",
+    name: Routes.Webshop,
+    params: {
+      slug: String,
+    },
+    show: "detail",
+    component: async () =>
+      (await import("../dashboard/webshop/WebshopOverview.vue"))
+        .default as unknown as ComponentOptions,
+    paramsToProps: ({ slug }: { slug: string }) => {
+      const webshop = visibleWebshops.value.find(
+        (shop) => Formatter.slug(shop.id) === slug
+      );
+
+      if (!webshop) {
+        throw new Error("Webshop not found");
+      }
+
+      selectWebshop(webshop);
+
+      return {
+        preview: webshop,
+      };
+    },
+    propsToParams(props) {
+      const webshop = props.preview as WebshopPreview | undefined;
+
+      if (!webshop) {
+        throw new Error("Missing preview (webshop)");
+      }
+
+      selectWebshop(webshop);
+
+      const slug = Formatter.slug(webshop.id);
+
+      return {
+        params: {
+          slug,
+        },
+      };
+    },
+    isDefault: {
+      properties: {
+        preview: visibleWebshops.value[0],
+      },
+    },
+  },
+  // add webshop
+  {
+    url: "nieuw",
+    name: Routes.AddWebshop,
+    show: "detail",
```

Comment:
```suggestion
    present: "popup",
```
Aanmaken van webshops zou ik in een popup houden voorlopig, om het consequent te houden